bootstrap/command/npm

```diff
@@ -18,7 +18,7 @@
 . $BACARDI_PATH/bootstrap/common/platform_info.sh
 
 if is_windows_platform; then
-  $(third_party_path)/node/npm.cmd $@
+  $(third_party_path)/node/npm.cmd --scripts-prepend-node-path true $@
```

Comment:
Why do we have to add scripts-prepend-node-path?

Reply:
actually do not need it. but it makes warning message.
```
WARN lifecycle The node binary used for scripts is C:\projects\bacardi\third_party\node\node.exe
but npm is using C:\projects\bacardi\THIRD_~1\node\node.exe itself.
Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.
```
and i removed it.